### PR TITLE
pallet meta & credit

### PR DESCRIPTION
**Pull request by smohan-dw**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -28,8 +28,7 @@ members = [
 	"pallets/space",
 	"pallets/schema",
 	"pallets/stream",
-	"pallets/authorities",
-	"pallets/builder",
-	"pallets/foundation",
+	"pallets/anchor",
```
Comment (amarts) [inline Cargo.toml]: the word 'anchor' itself is bit confusing when i just check this. (as anchor is still a name we use for 'create' method in many pallets already). Need to document this better.

File: pallets/schema/src/lib.rs [hunk 2]
```diff
@@ -151,77 +151,66 @@ pub mod pallet {
...
+				tx_schema.schema.controller.clone(),
+			)
+			.map_err(<Error<T>>::from)?;
+
+			SchemaDelegations::<T>::try_mutate(
+				tx_schema.identifier.clone(),
+				|ref mut delegation| {
+					ensure!(
+						delegation.len() + delegates.len() <= T::MaxSchemaDelegates::get() as usize,
+						Error::<T>::TooManyDelegates
```
Comment (amarts) [inline pallets/schema/src/lib.rs:196]: Any hardcoded number here for 'Too Many"? Can we change this number in the future?

Comment (amarts) [reply pallets/schema/src/lib.rs:196]: Applicable to every pallet where we use `TooManyDelegates` error.

**Review by amarts — COMMENTED** [review]
WIP, but wanted to send comments as I get them.

File: Cargo.toml (hunk 1 above)
Comment (amarts) [reply Cargo.toml]: `royalty_treasury`

**Comment by amarts** [comment]
not sure why, the debug build had lesser dependency than the `--release` build, and also didn't start finalizing the blocks in `--dev` mode. Now building the release mode to run some tests, will update here after testing with release build.

**Comment by amarts** [comment]
Ok, update on the above comment:

`cargo build --release` works. @smohan-dw confirmed with me that we support `--release` (with debug flags) and `--production` as two parameters for build. The default (`--debug`) is not supported in the build for now.

**Review by amarts — APPROVED** [review]

**Review by amarts — APPROVED** [review]

**Comment by smohan-dw** [comment]
The --release build has a subsection for --dev, which adds debug symbols to the --dev execution.

**Pull request merged**